apps/allocations/package.json

```diff
@@ -13,7 +13,8 @@
     "ganache-cli:test": "../shared/test-helpers/ganache-cli.sh",
     "lint": "solium --dir ./contracts",
     "prepublishOnly": "truffle compile --all",
-    "publish:http": "aragon apm publish --http localhost:2222 --http-served-from ./dist",
+    "publish:http": "npm run publish:http-copy-files && aragon apm publish --http localhost:2222 --http-served-from ./dist",
```

Comment:
What's the reason for using a different port for each app?

Reply:
Because with httpProvider we are simultaneously running 4 independent parcel server instances (actually 3 right now, because AddressBook is not), parcel is smart enough to figure if a instance is running and use a new random port, but since we need to previously know the port to tell aragon apm publish we define a different one in each app.